src/mame/misc/marywu.cpp

```diff
@@ -207,12 +207,12 @@ void marywu_state::marywu(machine_config &config)
 
 	/* sound hardware */
 	SPEAKER(config, "mono").front_center();
-	ay8910_device &ay1(AY8910(config, "ay1", XTAL(10'738'635) / 6)); /* should it be perhaps a fraction of the XTAL clock ? */
+	ay8910_device &ay1(AY8910(config, "ay1", XTAL(10'738'635) / 6)); /* should it be perhaps a fraction of the XTAL clock ? Also, an ALE pulse is skipped during each external data memory access, which will affect sound generation if the program does this.*/
```

Comment:
You've just added the division by 6, so you should remove the comment that had the question `"should it be perhaps a fraction of the XTAL clock ?"`